conjure-java-jackson-serialization/src/main/java/com/palantir/conjure/java/serialization/InstrumentedJsonFactory.java

```diff
@@ -0,0 +1,710 @@
+            delegate.clearCurrentToken();
+        }
+
+        @Override
+        public JsonToken getLastClearedToken() {
+            return delegate.getLastClearedToken();
+        }
+
+        @Override
+        public void overrideCurrentName(String name) {
+            delegate.overrideCurrentName(name);
+        }
+
+        @Override
+        public String getCurrentName() throws IOException {
+            return delegate.getCurrentName();
+        }
+
+        @Override
+        public String currentName() throws IOException {
+            return delegate.currentName();
+        }
+
+        @Override
+        public String getText() throws IOException {
+            return recordStringLength(delegate.getText());
+        }
+
+        @Override
+        public int getText(Writer writer) throws IOException, UnsupportedOperationException {
+            return delegate.getText(writer);
+        }
+
+        @Override
+        public char[] getTextCharacters() throws IOException {
+            return delegate.getTextCharacters();
+        }
+
+        @Override
+        public int getTextLength() throws IOException {
+            return delegate.getTextLength();
+        }
+
+        @Override
+        public int getTextOffset() throws IOException {
+            return delegate.getTextOffset();
+        }
+
+        @Override
+        public boolean hasTextCharacters() {
+            return delegate.hasTextCharacters();
+        }
+
+        @Override
+        public Number getNumberValue() throws IOException {
+            return delegate.getNumberValue();
+        }
+
+        @Override
+        public Number getNumberValueExact() throws IOException {
+            return delegate.getNumberValueExact();
+        }
+
+        @Override
+        public NumberType getNumberType() throws IOException {
+            return delegate.getNumberType();
+        }
+
+        @Override
+        public byte getByteValue() throws IOException {
+            return delegate.getByteValue();
+        }
+
+        @Override
+        public short getShortValue() throws IOException {
+            return delegate.getShortValue();
+        }
+
+        @Override
+        public int getIntValue() throws IOException {
+            return delegate.getIntValue();
+        }
+
+        @Override
+        public long getLongValue() throws IOException {
+            return delegate.getLongValue();
+        }
+
+        @Override
+        public BigInteger getBigIntegerValue() throws IOException {
+            return delegate.getBigIntegerValue();
+        }
+
+        @Override
+        public float getFloatValue() throws IOException {
+            return delegate.getFloatValue();
+        }
+
+        @Override
+        public double getDoubleValue() throws IOException {
+            return delegate.getDoubleValue();
+        }
+
+        @Override
+        public BigDecimal getDecimalValue() throws IOException {
+            return delegate.getDecimalValue();
```

Comment:
Do we need to track max length here as well (default limit `1000`)? Would we want to log warning if we exceed default limit?